src/main/java/racing/RacingCarOperator.java

```diff
@@ -0,0 +1,62 @@
+package racing;
+
+import java.util.Random;
+
+public class RacingCarOperator {
+    int numOfCar;
+    int numOfCycle;
+    int[] carLocation;
+
+    public RacingCarOperator(int[] racingInputInfo) {
+        this.numOfCar = racingInputInfo[0];
+        this.numOfCycle = racingInputInfo[1];
+        carLocation = new int[this.numOfCar];
+    }
+
+
+    public void Run() {
+        // 초기 위치 설정 (1)
+        InitCarLocation();
+
+        // 4-1) 시도 횟수가 0일 경우 다음 단계로 이동.
+        // 4-2) 시도 횟수가 1이상 일 경우 로직 1단계로 이동.
+        while (numOfCycle != 0) {
+            MoveCarLocation();
+            ShowCurrentCarLocation();
+
+            // 4) 시도 횟수 -= 1
+            numOfCycle--;
+        }
+    }
+
+    private void InitCarLocation() {
```

Comment:
코드 컨벤션 한번 확인 부탁 드립니다 🙇 
```suggestion
    private void initCarLocation() {
```

Reply:
코드 컨벤션을 매일 살펴보며 익히도록 하겠습니다.
수정하도록 하겠습니다!
감사합니다!

[메서드의 이름은 동사이어야 하며, 복합 단어일 경우 첫 단어는 소문자로 시작하고 그 이후에 나오는 단어의 첫 문자는 대문자로 사용해야 한다.]